### ISSUE TITLE
Including BITS and CHIPSEC as Coreboot Payloads

### ISSUE THREAD

**Issue by bluesteal** [osresearch/heads#1435]
Is it currently possible to include BITS and/or CHIPSEC to the Coreboot payload as binary utilities?

https://www.osfc.io/2018/talks/bits-and-chipsec-as-coreboot-payloads/

**Comment by tlaurion** [comment]
@piotr?

**Comment by pietrushnic** [comment]
Yes, it is possible. 3mdeb did that in the past, as presented in the above talk. We did that for validation purposes.

@bluesteal what would be your goal?

**Comment by bluesteal** [comment]
just want to be able to understand and analyze firmware in general, it would be a handy tool.

**Comment by pietrushnic** [comment]
@bluesteal I guess you could find some branches in our repos, but I doubt it can be compiled without any issues.

**Comment by bluesteal** [comment]
do you mean dependencies?

**Comment by pietrushnic** [comment]
Those branches are 5 years old:
* https://github.com/3mdeb/coreboot/tree/bits_chipsec_payload
* https://github.com/3mdeb/bits/pull/2
* https://github.com/3mdeb/chipsec - but it should be figure out from build process of coreboot if it is even included
